app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -11,9 +11,10 @@ def facebook
         #  puts intent
         # Don't create a new identity
       if intent == "sign_up"
+        puts intent
         # sign up flow, such as:
-        User.add_friends(@user.token, @user.id)
-        User.add_movies(@user.token, @user.id)
+        # @user.add_friends
```

Comment:
@sdsunjay you might also be able to add these as hooks in the user model, I'm not sure if devise has hooks built in for when a user signs up.

Reply:
I think there is hooks, in the users/registrations_controller.rb there is  `def after_sign_up_path_for(resource)`, which I could probably use instead.